message-queue-lib/src/cmr/message_queue/queue/memory_queue.clj

```diff
@@ -98,15 +104,18 @@
   ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
   queue-protocol/Queue
 
+  ;; parameters are 'this', queue-name, and message
   (publish-to-queue
-    [this queue-name msg]
+    [_ queue-name msg]
```

Comment:
Is publish-to-queue a protocol?  I am asking to see if we can get rid this in the parameters.  You could also use _this.